### PR TITLE
Ensures More Cards items don't break in to a single column

### PR DESCRIPTION
**Pull request by ryanml**
Fixes brave/brave-browser#12093

Ubuntu
Screen Shot 2020-10-12 at 10 26 43 PM

MacOS (No visible regressions)
Screen Shot 2020-10-12 at 10 27 35 PM

## Submitter Checklist:

- [x] Submitted a [ticket](https://github.com/brave/brave-browser/issues) for my issue if one did not already exist.
- [x] Used Github [auto-closing keywords](https://help.github.com/articles/closing-issues-via-commit-messages/) in the commit message.
- [ ] Added/updated tests for this change (for new code or code which already has tests).
- Verified that these changes build without errors on
  - [ ] Android
  - [ ] iOS
  - [ ] Linux
  - [x] macOS
  - [ ] Windows
- Verified that these changes pass automated tests (unit, browser, security tests) on
  - [ ] iOS
  - [ ] Linux
  - [ ] macOS
  - [ ] Windows
- [x] Verified that all lint errors/warnings are resolved (`npm run lint`, `npm run gn_check`)
- [x] Ran `git rebase master` (if needed).
- [ ] Ran `git rebase -i` to squash commits (if needed).
- [x] Tagged reviewers and labelled the pull request as needed.
- [ ] Requested a security/privacy review as needed.
- [ ] Added appropriate QA labels (`QA/Yes` or `QA/No`) to the associated issue
- [ ] Added appropriate release note labels (`release-notes/include` or `release-notes/exclude`) to the associated issue
- [ ] Public documentation has been updated as necessary. For instance:
  - [ ] https://github.com/brave/brave-browser/wiki/Deviations-from-Chromium-(features-we-disable-or-remove)
  - [ ] https://github.com/brave/brave-browser/wiki/Proxy-redirected-URLs
  - [ ] https://github.com/brave/brave-browser/wiki/Fingerprinting-Protections
  - [ ] https://github.com/brave/brave-browser/wiki/Brave%E2%80%99s-Use-of-Referral-Codes
  - [ ] https://github.com/brave/brave-browser/wiki/Custom-Headers
  - [ ] https://github.com/brave/brave-browser/wiki/Web-compatibility-issues-with-tracking-protection
  - [ ] https://github.com/brave/brave-browser/wiki/QA-Guide

## Test Plan:
Defined in issue

## Reviewer Checklist:

- [ ] New files have MPL-2.0 license header.
- [ ] Request a security/privacy review as needed.
- [ ] Adequate test coverage exists to prevent regressions 
- [ ] Verify test plan is specified in PR before merging to source

## After-merge Checklist:

- [ ] The associated issue milestone is set to the smallest version that the
  changes has landed on.
- [ ] All relevant documentation has been updated.

### CODE REVIEW

**Review by emerick — APPROVED** [review]
LGTM

File: components/brave_new_tab_ui/components/default/settings/index.ts [hunk 1]
```diff
@@ -197,7 +197,7 @@ export const SettingsSidebarButton = styled<SettingsSidebarButtonProps, 'button'
 export const SettingsFeatureBody = styled<{}, 'section'>('section')`
   padding: 10px 0;
   height: 360px;
-  overflow-y: auto;
+  overflow-y: overlay;
```
Comment (AlanBreck) [inline components/brave_new_tab_ui/components/default/settings/index.ts:200]: The `overlay` value for `overflow` is deprecated. Should we be concerned about future support?

Comment (ryanml) [reply components/brave_new_tab_ui/components/default/settings/index.ts:200]: I know it is in Firefox, but afaik there weren't plans to deprecate it in Webkit browsers. Is it on track for deprecation in Chrome?

Comment (AlanBreck) [reply components/brave_new_tab_ui/components/default/settings/index.ts:200]: Hmm... I'm not exactly sure if it's only deprecated in Firefox, or if it's being deprecated across the board. I'm not seeing it in the spec for overflow, though: https://www.w3.org/TR/css-overflow-3/#overflow-properties

Comment (ryanml) [reply components/brave_new_tab_ui/components/default/settings/index.ts:200]: https://developer.mozilla.org/en-US/docs/Web/CSS/overflow#Values

Comment (AlanBreck) [reply components/brave_new_tab_ui/components/default/settings/index.ts:200]: Looks like deprecation is in discussion, but I'm not seeing anything definitive. Don't think this should be a blocker. Sorry to distract.
https://bugs.chromium.org/p/chromium/issues/detail?id=554361#c19

**Comment by srirambv** [comment]
@ryanml  not sure if this is expected? There is no right padding on the add cards 
image
image

Adding a 20px margin to the right makes the card look correctly on Linux 
image

Also the Bitcoin.com widget and Crypto.com widget cards doesn't match up in y position
image

**Pull request merged**